### PR TITLE
Leetcode Scrapper

### PR DESCRIPTION
**Pull request by Hansika002**
Issue #259 
Created a Leetcode scrapper.
The code provided is a LeetCodeScraper class that uses Selenium and BeautifulSoup to scrape the rank and rating of a user on the LeetCode website. It initializes a headless Chrome browser using Selenium, navigates to the user's profile page, and retrieves the rank and rating from the HTML using BeautifulSoup. It returns the scraped data along with success or failure messages.

Please review it!!

### CODE REVIEW

File: src/scrape_up/leetcode/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from leetcode_scraper import LeetCodeScraper
+
+# Create an instance of the LeetCodeScraper
+scraper = LeetCodeScraper()
+
```
Comment (nikhil25803) [inline src/scrape_up/leetcode/__init__.py:5]: Hey @Hansika002, the `__init__.py` file is not well scripted. Look for other classes `__init__.py file for reference.

File: src/scrape_up/leetcode/leetcode_scraper.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            }
+        
+    def scrape_rating(self, username):
+        url = f"https://leetcode.com/{username}"
+        driver = webdriver.Chrome(options=self.chrome_options)
+        driver.get(url)
+
+        # Wait for the page to load
+        sleep(3)
+
```
Comment (nikhil25803) [inline src/scrape_up/leetcode/leetcode_scraper.py:54]: Add this class and the method in the `documentation.md` file also.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Requested some changes @Hansika002

**Comment by Hansika002** [comment]
I have made changes to the __init__.py file and also added the documentation in documentation.md file.
New commit - fba285b

Please check it!!

**Comment by nikhil25803** [comment]
Great work @Hansika002  !!
Do not forget to
+ Share your contribution on socials(twitter mainly), tag us and Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Pull request merged**